src/apps/loyalty/server/index.tsx

```diff
@@ -0,0 +1,46 @@
+import * as express from "express"
+import * as path from "path"
+import * as React from "react"
+import { renderToString } from "react-dom/server"
+import * as styleSheet from "styled-components/lib/models/StyleSheet"
+import renderPage from "./template"
+
+import Inquiries from "../containers/inquiries"
+import Login from "../containers/login"
+
+const app = express()
+
+app.use(express.static(path.resolve(__dirname)))
+
+app.get("/", (req, res) => {
+  // check if the user is logged in, if not redirect to /login
+  // else render /inquiries
+  res.redirect(req.baseUrl + "/inquiries")
+})
+
+app.get("/login", (req, res) => {
+  const html = renderToString(<Login />)
+  const styles = styleSheet.rules().map(rule => rule.cssText).join("\n")
+  res.send(renderPage({
+    styles,
+    html,
+    entrypoint: "/bundles/login.js",
+  }))
+})
+
+app.get("/inquiries", (req, res) => {
+  const user = (req as any).user
```

Comment:
Right, I didn't really define a dev flow as of now. I get what you're saying about referring to a method defined in another repo, I'll add the type def for `req.user`. 

Even though it's not the only way to do this, I think it makes sense to add `artsy-passport` to reaction-force and configure it with force's credentials. That way we can run it inside of force or as a standalone app. I'll set this up.